### PR TITLE
Add gcloud insert instance

### PR DESCRIPTION
**Pull request by victorursan**

### CODE REVIEW

File: cs-google-cloud/src/main/scala/io/cloudslang/content/gcloud/actions/compute/disks/DisksInsert.scala [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+import io.cloudslang.content.utils.BooleanUtilities.toBoolean
+import io.cloudslang.content.utils.NumberUtilities.toInteger
+import io.cloudslang.content.utils.OutputUtilities.{getFailureResultsMap, getSuccessResultsMap}
+import org.apache.commons.lang3.StringUtils.{EMPTY, defaultIfEmpty}
+
+/**
+  * Created by victor on 3/3/17.
+  */
+class DisksInsert {
+  @Action(name = "Insert Instance",
```
Comment (robert-sandor) [inline cs-google-cloud/src/main/scala/io/cloudslang/content/gcloud/actions/compute/disks/DisksInsert.scala]: i think you wanted to write "Insert Disks"

File: cs-google-cloud/src/main/scala/io/cloudslang/content/gcloud/actions/compute/instances/InstancesRestart.scala [hunk 2]
```diff
@@ -11,14 +11,13 @@ import io.cloudslang.content.gcloud.utils.Constants._
...
 import io.cloudslang.content.gcloud.utils.action.InputNames._
-import io.cloudslang.content.gcloud.utils.action.InputUtils
 import io.cloudslang.content.gcloud.utils.action.InputUtils._
 import io.cloudslang.content.gcloud.utils.action.InputValidator._
 import io.cloudslang.content.gcloud.utils.service.{GoogleAuth, HttpTransportUtils, JsonFactoryUtils}
 import io.cloudslang.content.utils.BooleanUtilities._
 import io.cloudslang.content.utils.NumberUtilities._
 import io.cloudslang.content.utils.OutputUtilities._
-import org.apache.commons.lang3.StringUtils._
+import org.apache.commons.lang3.StringUtils.{EMPTY, _}
```
Comment (robert-sandor) [inline cs-google-cloud/src/main/scala/io/cloudslang/content/gcloud/actions/compute/instances/InstancesRestart.scala]: this import don't seem right

File: cs-google-cloud/src/main/scala/io/cloudslang/content/gcloud/actions/compute/instances/InstancesStart.scala [hunk 3]
```diff
@@ -17,7 +17,7 @@ import io.cloudslang.content.gcloud.utils.service.{GoogleAuth, HttpTransportUtil
 import io.cloudslang.content.utils.BooleanUtilities._
 import io.cloudslang.content.utils.NumberUtilities._
 import io.cloudslang.content.utils.OutputUtilities._
-import org.apache.commons.lang3.StringUtils._
+import org.apache.commons.lang3.StringUtils.{EMPTY, _}
```
Comment (robert-sandor) [inline cs-google-cloud/src/main/scala/io/cloudslang/content/gcloud/actions/compute/instances/InstancesStart.scala]: this don't look ok either 😄

File: cs-google-cloud/src/main/scala/io/cloudslang/content/gcloud/actions/compute/instances/InstancesStop.scala [hunk 4]
```diff
@@ -8,16 +8,16 @@ import io.cloudslang.content.constants.OutputNames._
...
 import io.cloudslang.content.gcloud.utils.action.InputNames._
 import io.cloudslang.content.gcloud.utils.action.InputUtils._
 import io.cloudslang.content.gcloud.utils.action.InputValidator._
-import io.cloudslang.content.gcloud.utils.action.{DefaultValues, InputUtils}
 import io.cloudslang.content.gcloud.utils.service.{GoogleAuth, HttpTransportUtils, JsonFactoryUtils}
 import io.cloudslang.content.utils.BooleanUtilities._
 import io.cloudslang.content.utils.NumberUtilities._
 import io.cloudslang.content.utils.OutputUtilities._
-import org.apache.commons.lang3.StringUtils._
+import org.apache.commons.lang3.StringUtils.{EMPTY, _}
```
Comment (robert-sandor) [inline cs-google-cloud/src/main/scala/io/cloudslang/content/gcloud/actions/compute/instances/InstancesStop.scala]: this too

File: cs-google-cloud/src/main/scala/io/cloudslang/content/gcloud/actions/compute/disks/DisksDelete.scala [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+    }
+
+    val proxyPort = toInteger(proxyPortStr)
+    val prettyPrint = toBoolean(prettyPrintStr)
+
+    try {
+      val httpTransport = HttpTransportUtils.getNetHttpTransport(proxyHostOpt, proxyPort, proxyUsernameOpt, proxyPassword)
+      val jsonFactory = JsonFactoryUtils.getDefaultJacksonFactory
+
+      val credential = GoogleAuth.fromAccessToken(accessToken)
```
Comment (pintea-eugen) [inline cs-google-cloud/src/main/scala/io/cloudslang/content/gcloud/actions/compute/disks/DisksDelete.scala]: should these be grouped?

File: cs-google-cloud/src/main/scala/io/cloudslang/content/gcloud/actions/compute/disks/DisksInsert.scala [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+        diskType = diskType,
+        diskDescription = diskDescription,
+        licensesList = licensesList,
+        licensesDel = licensesDel,
+        diskSize = diskSize)
+
+      val bootDisk = DiskService.insert(httpTransport, jsonFactory, credential, projectId, zone, computeDisk)
+      val resultString = if (prettyPrint) bootDisk.toPrettyString else bootDisk.toString
+
+      getSuccessResultsMap(resultString)
```
Comment (pintea-eugen) [inline cs-google-cloud/src/main/scala/io/cloudslang/content/gcloud/actions/compute/disks/DisksInsert.scala]: + ZoneOperations

**Pull request merged**